app/views/targets/index.json.jbuilder

```diff
@@ -0,0 +1 @@
+json.targets @targets, partial: 'info', as: :target
```

Comment:
here I think is probably the issue we are having with the `render`.
I will try to do something like: 
```
json.targets @targets.each do |target|
    json.target do 
        json.partial! 'info', target: target 
   end
end 
```